examples/keras_io/tensorflow/vision/bit.py

```diff
@@ -0,0 +1,305 @@
+"""
+Title: Image Classification using BigTransfer (BiT)
+Author: [Sayan Nath](https://twitter.com/sayannath2350)
+Date created: 2021/09/24
+Last modified: 2021/09/24
+Description: BigTransfer (BiT) State-of-the-art transfer learning for image classification.
+Accelerator: GPU
+"""
+
+"""
+## Introduction
+
+BigTransfer (also known as BiT) is a state-of-the-art transfer learning method for image
+classification. Transfer of pre-trained representations improves sample efficiency and
+simplifies hyperparameter tuning when training deep neural networks for vision. BiT
+revisit the paradigm of pre-training on large supervised datasets and fine-tuning the
+model on a target task. The importance of appropriately choosing normalization layers and
+scaling the architecture capacity as the amount of pre-training data increases.
+
+BigTransfer(BiT) is trained on public datasets, along with code in
+[TF2, Jax and Pytorch](https://github.com/google-research/big_transfer). This will help anyone to reach
+state of the art performance on their task of interest, even with just a handful of
+labeled images per class.
+
+You can find BiT models pre-trained on
+[ImageNet](https://image-net.org/challenges/LSVRC/2012/index) and ImageNet-21k in
+[TFHub](https://tfhub.dev/google/collections/bit/1) as TensorFlow2 SavedModels that you
+can use easily as Keras Layers. There are a variety of sizes ranging from a standard
+ResNet50 to a ResNet152x4 (152 layers deep, 4x wider than a typical ResNet50) for users
+with larger computational and memory budgets but higher accuracy requirements.
+
+![](https://i.imgur.com/XeWVfe7.jpeg)
+Figure: The x-axis shows the number of images used per class, ranging from 1 to the full
+dataset. On the plots on the left, the curve in blue above is our BiT-L model, whereas
+the curve below is a ResNet-50 pre-trained on ImageNet (ILSVRC-2012).
+"""
+
+"""
+## Setup
+"""
+
+import keras_core as keras
```

Comment:
Please set the backend env variable to tensorflow before import Keras Core, since the example is TF only